### PR TITLE
[staging-17.0] Merge Master up to e87eed7a9

### PR DESCRIPTION
**Pull request by darcywong00**
4 of x merging master from e87eed7a9 to staging-17.0

### CODE REVIEW

File: release/sil/sil_devanagari_typewriter/source/sil_devanagari_typewriter.kps [hunk 1]
```diff
@@ -1,175 +1,199 @@
...
+      <FileType>.md</FileType>
+    </File>
+  </Files>
+  <Keyboards>
+    <Keyboard>
+      <Name>Devanagari Typewriter (SIL)</Name>
+      <ID>sil_devanagari_typewriter</ID>
+      <Version>1.1.1</Version>
+      <OSKFont>..\..\..\shared\fonts\sil\annapurna_sil_nepal\AnnapurnaSILNepal-Regular.ttf</OSKFont>
+      <DisplayFont>..\..\..\shared\fonts\sil\annapurna_sil_nepal\AnnapurnaSILNepal-Bold.ttf</DisplayFont>
```
Comment (mcdurdin) [inline release/sil/sil_devanagari_typewriter/source/sil_devanagari_typewriter.kps:143]:
```suggestion
      <Version>1.1.1</Version>
      <OSKFont>..\..\..\shared\fonts\sil\annapurna_sil_nepal\AnnapurnaSILNepal-Bold.ttf</OSKFont>
      <DisplayFont>..\..\..\shared\fonts\sil\annapurna_sil_nepal\AnnapurnaSILNepal-Regular.ttf</DisplayFont>
```

Looks like you swapped these?

**Review by mcdurdin — APPROVED** [review]
Not sure if that font change is something in master or isolated to this PR. If it's in master, then match that!

File: release/sil/sil_devanagari_typewriter/source/sil_devanagari_typewriter.kps (hunk 1 above)
Comment (darcywong00) [reply release/sil/sil_devanagari_typewriter/source/sil_devanagari_typewriter.kps:143]: Hmm - The changes came from
9cdb3b794147b1b3c4be4e74c1dac89f43597467

But I'm confused why master shows
https://github.com/keymanapp/keyboards/blob/e2badecea87d7a13f4678078ea48fc524256a947/release/sil/sil_devanagari_typewriter/source/sil_devanagari_typewriter.kps#L128-L130

Comment (mcdurdin) [reply release/sil/sil_devanagari_typewriter/source/sil_devanagari_typewriter.kps:143]: OK, so looks like @LornaSIL may have inadvertently swapped the OSKFont and DisplayFont entries in PR #2474. Let's treat this as a separate issue to resolve once the merge is complete.

Comment (darcywong00) [reply release/sil/sil_devanagari_typewriter/source/sil_devanagari_typewriter.kps:143]: To be handled in #2760

**Pull request merged**